### PR TITLE
Added Models

### PR DESCRIPTION
**Pull request by ConnorMRyan**
Added Mareo's model.

### CODE REVIEW

**Comment by mareo1997** [comment]
Conner I already made the models yesterday

File: src/main/java/com/revature/backend/model/Analysis_Item.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
+package com.revature.backend.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "analysisItems")
+public class Analysis_Item {
```
Comment (RevatureGentry) [inline src/main/java/com/revature/backend/model/Analysis_Item.java]: Please change this to PascalCase, not Upper_Snake_Case.

File: src/main/java/com/revature/backend/model/Analysis_Item.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+    public boolean equals(final Object o) {
+        if (o == this) return true;
+        if (!(o instanceof Analysis_Item)) return false;
+        final Analysis_Item other = (Analysis_Item) o;
+        if (!other.canEqual((Object) this)) return false;
+        if (this.getId() != other.getId()) return false;
+        final Object this$content = this.getContent();
+        final Object other$content = other.getContent();
+        if (this$content == null ? other$content != null : !this$content.equals(other$content)) return false;
```
Comment (RevatureGentry) [inline src/main/java/com/revature/backend/model/Analysis_Item.java]: Is this supposed to be `this$content == null`?

File: src/main/java/com/revature/backend/model/Analysis_Type.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.revature.backend.model;
+
+public enum Analysis_Type {
```
Comment (RevatureGentry) [inline src/main/java/com/revature/backend/model/Analysis_Type.java]: PascalCase, not Upper_Snake_Case.

**Comment by ConnorMRyan** [comment]
That's why I implemented yours and didn't create my own. You made yours before we had the file structure set, so they needed migrated. We'd have a merge issue if we just tried to merge your branch into main.

**Review by RevatureGentry — CHANGES_REQUESTED** [review]
All in all this LGTM, but please use proper Java notation. Use PascalCase for class names, and camelCase for fields. 
Also please investigate if you intended to invoke `this$something` in `Manager.java` and `Analysis_Item.java` instead of `this.something`

File: src/main/java/com/revature/backend/model/Analysis_Item.java (hunk 1 above)
Comment (sophiagavrila) [reply src/main/java/com/revature/backend/model/Analysis_Item.java]: @ConnorMRyan see comment from William above regarding `PascalCase`

File: src/main/java/com/revature/backend/model/Analysis_Item.java (hunk 2 above)
Comment (ConnorMRyan) [reply src/main/java/com/revature/backend/model/Analysis_Item.java]: That's a remnant of delombaking, I'll see what they were meaning to do

File: src/main/java/com/revature/backend/model/Associate.java [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+@Table(name = "associate")
+public class Associate {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+
+    @Column(name = "salesforce_id")
+    private String salesforce_id;
```
Comment (RevatureGentry) [inline src/main/java/com/revature/backend/model/Associate.java]: Please fix all the fields, getters, and setters in this class to also be camelCase.

File: src/main/java/com/revature/backend/model/AssociateStatus.java [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package com.revature.backend.model;
+
+public enum AssociateStatus {
+    staging, project
```
Comment (RevatureGentry) [inline src/main/java/com/revature/backend/model/AssociateStatus.java]: Values of enums are also typically in all uppercase.

```java
public enum AssociateStatus {
  STAGING,
  PROJECT
}
```

**Review by RevatureGentry — CHANGES_REQUESTED** [review]
Please convert all fields, getters, and setters from snake_case to pascalCase.

Also, @ConnorMRyan, after looking at this a little further, let's change `SWOT` to `Swot`. Looking at `AnalysisItem`, the `SWOT__id` looks a little jarring to me. What do you think?

File: src/main/java/com/revature/backend/model/Associate.java (hunk 4 above)
Comment (ConnorMRyan) [reply src/main/java/com/revature/backend/model/Associate.java]: I agree SWOT__ID does look strange, SwotID?

Comment (ConnorMRyan) [reply src/main/java/com/revature/backend/model/Associate.java]: I believe I got all of them.

**Pull request merged**